### DISCUSSION TITLE
Tycho pomless: How to define the repository name?

### DISCUSSION THREAD

**Discussion by jpfeuffer** [eclipse-tycho/tycho#2186] [Q&A] [answered]
Hi!
I am building an update site with tycho-pomless, based on the category.xml.
How can I specify the name that will be included in the content.xml of the resulting p2 repo/updatesite? It is used to display the update site in the list of update sites in my product.

Currently it always is `<repository name='[updatesite] ${updatesiteartifactid}.eclipse-repository' type='org.eclipse.equinox.internal.p2.metadata.repository.LocalMetadataRepository' version='1'>`.

I cannot even find a full schema for the category.xml to see which elements are allowed.

**Comment by laeubi** [comment, marked as answer]
I can guess it uses the project name, you can override it like described here:
https://github.com/eclipse-tycho/tycho/wiki/Tycho-Pomless#overwrite-group-and-artifact-ids

**Reply by jpfeuffer** [reply]
Cool. However the documentation only shows a case where you put the build.properties into the overall plugins/features/... folder. Can I also put it into the subfolder of a single unit, i.e. besides the category.xml in my case?

I think I will try tomorrow.

**Reply by laeubi** [reply]
It can be any (module) folder.
